Wikipedia/Code/NSURLComponents+WMFLinkParsing.m

```diff
@@ -105,4 +105,32 @@ - (NSString *)wmf_fragment {
     return [self.fragment precomposedStringWithCanonicalMapping];
 }
 
+- (nullable NSString *)wmf_eventLoggingLabel {
+    NSString *eventLoggingLabel = nil;
```

Comment:
Would `wmf_valueForQueryKey` be useful here? (and `wmf_urlWithoutQueryKey:` for removing it?)